src/olympia/addons/migrations/0029_auto_20210118_1527.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.17 on 2021-01-18 15:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('addons', '0028_auto_20210114_1347'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='category',
+            name='addons',
+        ),
+        # migrations.RemoveConstraint(
+        #     model_name='addoncategory',
+        #     name='addon_id',
+        # ),
+        # migrations.RemoveIndex(
+        #     model_name='addoncategory',
+        #     name='category_addon_idx',
+        # ),
+        migrations.RemoveField(
+            model_name='addon',
+            name='categories',
+        ),
+        migrations.AlterField(
+            model_name='addoncategory',
+            name='category',
+            field=models.SmallIntegerField(),
+        ),
+        migrations.RenameField(
+            model_name='addoncategory',
+            old_name='category',
+            new_name='category_id',
+        ),
+        migrations.AddIndex(
+            model_name='addoncategory',
+            index=models.Index(fields=['category_id', 'addon'], name='category_id_addon_idx'),
+        ),
+        migrations.AddConstraint(
+            model_name='addoncategory',
+            constraint=models.UniqueConstraint(fields=('addon', 'category_id'), name='category_id_addon_id'),
+        ),
+        migrations.DeleteModel(
```

Comment:
Wouldn't that fail with the old code as soon as the migration is applied too ?

Reply:
Actually yeah, this step would break lookups during the deploy.  If we opt for leaving the rest of the migration 2021-01-28 in  this would have to be in later tag.

Reply:
It's a big mess (in pre-django-migration days we could just drop the constraint and that would be all that was needed!)

Reply:
The alternative is we add a _new_ column for the category id, and copy the value from `category_id` during the migration.  But that seemed wasteful (and probably really slow)